modules/new_serverpod_auth/serverpod_auth_idp/serverpod_auth_idp_server/lib/src/providers/email/util/session_extension.dart

```diff
@@ -4,9 +4,10 @@ import 'package:serverpod/serverpod.dart';
 extension SessionExtension on Session {
   /// Returns the client's IP address, or empty string in case it could not be
   /// determined.
-  String get remoteIpAddress {
+  IPAddress? get remoteIpAddress {
```

Comment:
Now actually returns the remote IPAddress of the TCP connection. This extension is only used Email IDP and only for logging. I believe this should be changed, but let us discuss and do so in a separate PR: